src/main/java/org/folio/service/PatronBlocksService.java

```diff
@@ -200,27 +234,34 @@ private BlocksCalculationContext addActionBlocksByLimitAndConditionToContext(
       Boolean.TRUE.equals(patronBlockCondition.getBlockRenewals()),
       Boolean.TRUE.equals(patronBlockCondition.getBlockRequests()));
 
-    return ctx.withCurrentActionBlocks(
+    BlocksCalculationContext result = ctx.withCurrentActionBlocks(
       ActionBlocks.and(actionBlocksByLimit, actionBlocksByCondition));
+    log.info("addActionBlocksByLimitAndConditionToContext:: result: {}",
+      () -> logAsJson(result));
+    return result;
   }
 
   private AutomatedPatronBlock createBlockForLimit(BlocksCalculationContext ctx) {
+    log.debug("createBlockForLimit:: parameters ctx: {}", () -> logAsJson(ctx));
     if (ctx.currentPatronBlockCondition == null || ctx.currentActionBlocks == null) {
       ctx.logFailedValidationError("createBlockForLimit");
       return null;
     }
 
-    return new AutomatedPatronBlock()
+    AutomatedPatronBlock result = new AutomatedPatronBlock()
       .withPatronBlockConditionId(ctx.currentPatronBlockCondition.getId())
       .withBlockBorrowing(ctx.currentActionBlocks.getBlockBorrowing())
       .withBlockRenewals(ctx.currentActionBlocks.getBlockRenewals())
       .withBlockRequests(ctx.currentActionBlocks.getBlockRequests())
       .withMessage(ctx.currentPatronBlockCondition.getMessage());
+    log.info("createBlockForLimit:: result: {}", () -> logAsJson(result));
+    return result;
   }
 
   @With
   @AllArgsConstructor
   @NoArgsConstructor(force = true)
+  @Getter
```

Comment:
Is this actually used somewhere?

Reply:
Yes, without getters JsonObject can't map this object to JSON when it's logged